apis/policies/v1/policyserver_types_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestBasicCRD(t *testing.T) {
+	capabilities := corev1.Capabilities{
+		Add:  []corev1.Capability{corev1.Capability("ALL")},
+		Drop: []corev1.Capability{corev1.Capability("ALL")},
+	}
+	var user, group int64
+	user = 1000
+	group = 2000
+	privileged := true
+	containerSecurity := corev1.SecurityContext{
+		Privileged:   &privileged,
+		Capabilities: &capabilities,
+	}
+	podSecurity := corev1.PodSecurityContext{
+		RunAsUser:  &user,
+		RunAsGroup: &group,
+	}
+	_ = PolicyServerSpec{
```

Comment:
This is a bit strange test, is it checking that we can define an object like the one described inside of `policyserver_types.go`?

If that's the case, it feels a bit like testing the code generation capabilities of kubebuilder. I don't think this is worth the effort, but I'm not against leaving this code around

Reply:
Agree here.

Reply:
File removed. ;)